public/style.css

```diff
@@ -9,6 +9,31 @@
   font-family: 'Inconsolata', monospace !important;
 }
 
+.listUI{
+  background: #101010;
+  color:white;
+  width:100%;
+  border: 1px solid black;
+  padding:5px;
+  margin: 15px;
+  border-radius:3px;
+  box-shadow: 2px 2px white !important;
```

Comment:
use `uk-box-shadow`-* instead, using `!important` is not a good practice.
https://getuikit.com/docs/utility#box-shadow